client/stylesheets/_toCat.scss

```diff
@@ -5599,4 +5600,128 @@ textarea {
     border-radius: 4px !important;
 }
 
+/*~~~~~~~~~ Default Profile Message ~~~~~~~~~*/
+
+.defaultProfileDiv {
+    text-align: center;
+    margin-top: 120px !important;
+}
+
+.defaultProfileDiv h2 {
+    font-weight: 400 !important;
+    color: #555 !important;
+    vertical-align: middle !important;
+}
+
+.defaultProfileDiv h2 i {
```

Comment:
Merge rule `.defaultProfileDiv h2 i` with rule on line 5605<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector `defaultProfileDiv` should be written in lowercase with hyphens